tests/test_parse_load_args.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+import unittest
+import uuid
+from typing import List, Optional
+
+import mkosi
+import mkosi.backend as backend
+from mkosi.backend import MkosiArgs, MkosiException, Distribution
+
+
+def load_args_wrapper(argv: Optional[List[str]] = None) -> MkosiArgs:
+    return mkosi.load_args(mkosi.parse_args(argv)["default"])
+
+
+class TestMkosiMethods(unittest.TestCase):
+
+    def test_parse_load_verb(self):
+        self.assertIsInstance(load_args_wrapper([]), MkosiArgs)
+        self.assertEqual(load_args_wrapper(["build"]).verb, "build")
+        self.assertEqual(load_args_wrapper(["clean"]).verb, "clean")
+        with self.assertRaises(SystemExit):
+            load_args_wrapper(["help"])
+        self.assertEqual(load_args_wrapper(["genkey"]).verb, "genkey")
+        self.assertEqual(load_args_wrapper(["bump"]).verb, "bump")
+        self.assertEqual(load_args_wrapper(["serve"]).verb, "serve")
+        self.assertEqual(load_args_wrapper(["build"]).verb, "build")
+        self.assertEqual(load_args_wrapper(["shell"]).verb, "shell")
+        self.assertEqual(load_args_wrapper(["boot"]).verb, "boot")
+        self.assertEqual(load_args_wrapper(["qemu"]).verb, "qemu")
+        #self.assertEqual(load_args_wrapper(["ssh"]), self.test_MkosiArgs)
+        with self.assertRaises(SystemExit):
+            load_args_wrapper(["invalid"])
+
+    def test_os_distribution(self):
+        self.assertEqual(load_args_wrapper(["-d", "fedora"]).distribution, Distribution.fedora)
+        self.assertEqual(load_args_wrapper(["-d", "debian"]).distribution, Distribution.debian)
+        self.assertEqual(load_args_wrapper(["-d", "ubuntu"]).distribution, Distribution.ubuntu)
+        self.assertEqual(load_args_wrapper(["-d", "arch"]).distribution, Distribution.arch)
+        self.assertEqual(load_args_wrapper(["-d", "opensuse"]).distribution, Distribution.opensuse)
+        self.assertEqual(load_args_wrapper(["-d", "mageia"]).distribution, Distribution.mageia)
+        self.assertEqual(load_args_wrapper(["-d", "centos"]).distribution, Distribution.centos)
+        self.assertEqual(load_args_wrapper(["-d", "centos_epel"]).distribution, Distribution.centos_epel)
+        self.assertEqual(load_args_wrapper(["-d", "clear"]).distribution, Distribution.clear)
+        self.assertEqual(load_args_wrapper(["-d", "photon"]).distribution, Distribution.photon)
+        self.assertEqual(load_args_wrapper(["-d", "openmandriva"]).distribution, Distribution.openmandriva)
+        self.assertEqual(load_args_wrapper(["-d", "rocky"]).distribution, Distribution.rocky)
+        self.assertEqual(load_args_wrapper(["-d", "rocky_epel"]).distribution, Distribution.rocky_epel)
+        self.assertEqual(load_args_wrapper(["-d", "alma"]).distribution, Distribution.alma)
+        self.assertEqual(load_args_wrapper(["-d", "alma_epel"]).distribution, Distribution.alma_epel)
```

Comment:
This isn't robust to new distribution being added. Instead, we should loop over the members of the `Distribution` enum so that any new additions are tested as well.

```
for d in Distribution:
    self.assertEqual(load_args_wrapper(["-d", d.name]).distribution, d)
```